### PR TITLE
adding adx #99

### PR DESCRIPTION
**Pull request by nachoalonsoportillo**
# Description

adding adx support

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #99

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Unit tests passing

### CODE REVIEW

File: internal/renderers/services.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func renderServices(f *excelize.File, data ReportData) {
 			setHyperLink(f, "Services", 12, currentRow)
 		}
 
-		configureSheet(f, "Services", heathers, currentRow)
```
Comment (cmendible) [inline internal/renderers/services.go:68]: Ohhhhhh dude I knew there was something wrong here!!!! Thanks for fixing this!

**Review by cmendible — CHANGES_REQUESTED** [review]
Please add `DataExplorerScanner` to the services list in `cmd\azqr\scan.go` and `cmd\azqr\rules.go`

File: internal/renderers/services.go (hunk 1 above)
Comment (nachoalonsoportillo) [reply internal/renderers/services.go:68]: It was nothing terrible, since it worked great and wasn't exposed to the end user anyways.

**Review by cmendible — APPROVED** [review]
LGTM

**Pull request merged**